src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int numberBall = 3;
```

Comment:
Don't forget about the naming convention for constants. See [here](https://stackoverflow.com/questions/12519475/java-constant-variable-naming-convention).

Also, let's rename it to BALLS_COUNT.